dockerfiles/init/modules/openshift/files/scripts/ocp.sh

```diff
@@ -148,15 +158,46 @@ run_ocp() {
 }
 
 deploy_che_to_ocp() {
-    #Repull init image only if IMAGE_PULL_POLICY is set to Always
-    if [ $IMAGE_PULL_POLICY == "Always" ]; then
-        docker pull "$IMAGE_INIT"
+    #Only generate scripts and config files if deploy_che.sh does not exist in same folder
+    if [ ! -f deploy_che.sh ]; then
```

Comment:
the idea was to never use local scripts and always use generated because some files required to deploy may be generated only with no defaults. technically local script works for now but it should not be used in that way. especially taking in account fact that Mario works on integration this to CLI